<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MessagesView</class>
 <widget class="QWidget" name="MessagesView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>835</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="messageFWindow" stretch="1,1,1,20">
     <property name="spacing">
      <number>1</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="msgDropdownLayout" stretch="1,1,1,1,0">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color: black;</string>
         </property>
         <property name="text">
          <string>Past</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="msgHoursSpinBox">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="alignment">
          <set>Qt::AlignJustify|Qt::AlignVCenter</set>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::PlusMinus</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="msgHoursLabel">
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color: black;</string>
         </property>
         <property name="text">
          <string>Hours |</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="styleSheet">
          <string notr="true">color: black;</string>
         </property>
         <property name="text">
          <string>Type:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="msgTypeDropdown">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <item>
          <property name="text">
           <string>TRAFFIC_ANNOUNCEMENT</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>EXEMPTED_TRANSPORT</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>WEIGHT_RESTRICTION</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ROAD_WORK</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="msgButtonLayout" stretch="1,1,2">
       <item>
        <widget class="QPushButton" name="msgShowButton">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #2196F3;
color: white;</string>
         </property>
         <property name="text">
          <string>Show</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="msgSaveButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton:disabled{background-color: #D9D9D9; color: white;} QPushButton:enabled{background-color: #2196F3; color: white;}</string>
         </property>
         <property name="text">
          <string>Save data</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="msgCompareButton">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton:disabled{background-color: #D9D9D9; color: white;} QPushButton:enabled{background-color: #2196F3; color: white;}</string>
         </property>
         <property name="text">
          <string>Load saved data</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QFrame" name="msgSaveSuccessNoti">
       <property name="styleSheet">
        <string notr="true">background-color: #2521F3;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="4,1">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Messages successfully saved!</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="msgCloseNotiButton">
          <property name="styleSheet">
           <string notr="true">background-color: #2196F3;</string>
          </property>
          <property name="text">
           <string>Close this dialog</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QScrollArea" name="msgScrollArea">
       <property name="styleSheet">
        <string notr="true">background-color: white;</string>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <widget class="QWidget" name="msgScrollWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>476</width>
          <height>711</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_5" rowstretch="1,20">
         <property name="leftMargin">
          <number>12</number>
         </property>
         <item row="1" column="0">
          <layout class="QVBoxLayout" name="msgCurrentBoxesLayout">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
          </layout>
         </item>
         <item row="1" column="1">
          <layout class="QVBoxLayout" name="msgComparingBoxesLayout"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="msgCurrentTitleLabel">
           <property name="styleSheet">
            <string notr="true">color: black;
font-weight: 500;
font-size: 18px;</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignHCenter|Qt::AlignTop</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="msgComparingTitleLabel">
           <property name="styleSheet">
            <string notr="true">color: black;
font-weight: 500;
font-size: 18px;</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignHCenter|Qt::AlignTop</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
